### PR TITLE
Stack update `v2024W30` for `aws-linux-baremetal`

### PR DESCRIPTION
**Pull request by bitrise-infrabot**

### CODE REVIEW

File: content/changelogs/aws/aws-linux-baremetal.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+{{< hint info >}}
+Learn more [how to get notified of updates]({{% ref "/tips/Get notified" %}}).
+{{< /hint >}}
+
+## Updates
+
+### Stack update `2024w30`
+
+- Bitrise Tooling Update
+- Java 17 added as default
```
Comment (ofalvai) [inline content/changelogs/aws/aws-linux-baremetal.md:18]:
```suggestion
- JDK 17 is the new default version (JDK 11 remains installed)
```

File: content/changelogs/aws/aws-linux-baremetal.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+{{< hint info >}}
+Learn more [how to get notified of updates]({{% ref "/tips/Get notified" %}}).
+{{< /hint >}}
+
+## Updates
+
+### Stack update `2024w30`
+
+- Bitrise Tooling Update
```
Comment (ofalvai) [inline content/changelogs/aws/aws-linux-baremetal.md:17]: What does this mean?

Comment (ofalvai) [reply content/changelogs/aws/aws-linux-baremetal.md:17]: Emulator got a big bump, it should be noted

**Review by ofalvai — CHANGES_REQUESTED** [review]

File: content/changelogs/aws/aws-linux-baremetal.md (hunk 2 above)
Comment (pgyula) [reply content/changelogs/aws/aws-linux-baremetal.md:17]: Bitrise Cli, DEN agent

Comment (ofalvai) [reply content/changelogs/aws/aws-linux-baremetal.md]: But we never mention this, and the CLI is updated on the fly anyways

**Review by ofalvai — APPROVED** [review]

**Pull request merged**